src/command/execute_program.rs

```diff
@@ -150,13 +150,21 @@ fn build_result_archive(res: ResultId) -> Result<(), std::io::Error> {
         }
     }
 
-    let _ = Command::new("zip")
-        .arg("-0")
+    let path_to_res = format!("./archives/{}/results", res.program_id);
+    let result = format!("{}", res.timestamp);
+    let path_to_log = String::from("../../../data");
+    let log = format!("{}_{}.log", res.program_id, res.timestamp);
+    let _ = Command::new("tar")
+        .arg("-cf")
         .arg(out_path)
-        .arg("--junk-paths")
+        .arg("--exclude")
         .arg("log")
-        .arg(res_path)
-        .arg(log_path)
+        .arg("-C")
+        .arg(path_to_res)
+        .arg(result)
+        .arg("-C")
+        .arg(path_to_log)
+        .arg(log)
```

Comment:
I would prefer to have the file paths specified directly, i.e. without changing directories with `-C`, just to reduce the used strings a bit and make it more obvious what is included